PERFTEST.PS1

```diff
@@ -8,11 +8,11 @@ Param(
     
     [ValidateSet("Default", "Azure", "Detail", "Max")]
     [parameter(Mandatory=$false)] [string] $Config = "Default",
-    [parameter(Mandatory=$false)]  [Array] $ToolList = @("ntttcp", "latte", "cps", "ctstraffic")
+    [parameter(Mandatory=$false)]  [Array] $ToolList = @("ntttcp", "latte", "l4ping", "cps", "ctstraffic")
 )
 
 $scriptName = $MyInvocation.MyCommand.Name 
-$SupportedTools = @("ntttcp", "latte", "cps", "ctstraffic")
+$SupportedTools = @("ntttcp", "latte", "l4ping", "cps", "ctstraffic")
```

Comment:
nit: use single quotes

Reply:
Leaving as is based on offline discussion.